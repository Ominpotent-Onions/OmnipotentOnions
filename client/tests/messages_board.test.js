'use strict'

import React from 'react';
import MessageBoard from '../src/components/messages_board';
import renderer from 'react-test-renderer';

const messages = [
  {
    id: 1,
    user: 'Shi-Hao',
    text: 'Hello friends!'
  },
  {
    id: 2,
    user: 'Peter',
    text: 'What\'s up!'
  },
  {
    id: 3,
    user: 'Dylan',
    text: 'I got the app deployed on Digital Ocean!' 
  },
  {
    id: 4,
    user: 'Gideon',
    text: 'That\'s awesome! Here, have some spam!'  
  }
];

it('renders correctly', () => {
  const tree = renderer.create(<MessageBoard 
    messages={messages}
  />).toJSON();
  expect(tree).toMatchSnapshot();
})