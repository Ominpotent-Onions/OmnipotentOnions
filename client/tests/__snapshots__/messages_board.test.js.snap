// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="ui segment"
  >
    <h4>
      Shi-Hao
    </h4>
    <div>
      Hello friends!
    </div>
  </div>
  <div
    className="ui segment"
  >
    <h4>
      Peter
    </h4>
    <div>
      What's up!
    </div>
  </div>
  <div
    className="ui segment"
  >
    <h4>
      Dylan
    </h4>
    <div>
      I got the app deployed on Digital Ocean!
    </div>
  </div>
  <div
    className="ui segment"
  >
    <h4>
      Gideon
    </h4>
    <div>
      That's awesome! Here, have some spam!
    </div>
  </div>
</div>
`;
